components/mailgun/sources/new-bounce/new-bounce.js

```diff
@@ -0,0 +1,50 @@
+const common = require("../common-webhook");
+const { mailgun } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-bounce",
```

Comment:
These `keys` should read `mailgun-new-bounce` (keys should always be preceded by the app name)

Reply:
ah! similarly, changes to reddit will need to be made too.